Resources/Settings.bundle/Root.inApp.plist

```diff
@@ -33,6 +13,14 @@
 			<string>SETTINGS_DARKTHEME</string>
 			<key>Type</key>
 			<string>PSToggleSwitchSpecifier</string>
+			<key>IASKSubtitle</key>
+			<string>Change app colours</string>
```

Comment:
How do I go about adding a new string that needs to be localized??

Reply:
You need to add constants and their plain text counter-part to the en.lproj's "Root.strings" within the Settings.bundle.